changelog.md

```diff
@@ -4,6 +4,11 @@
 
 - Added the `cai_causal_graph.causal_graph.CausalGraph.replace_edge` method to the
   `cai_causal_graph.causal_graph.CausalGraph` class, which allows you to replace an existing edge with a new one.
+- Added the `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.get_nodes_for_variable` method to the
```

Comment:
Missing new `get_nodes_at_lag` method in changelog.

Reply:
This isn't a new method, I just updated the docsting to be consistent with my method

Reply:
My bad. Completely missed that the `def` line wasn't new. Resolving this comment.